src/main/java/org/folio/service/auth/SystemUserAuthService.java

```diff
@@ -0,0 +1,287 @@
+package org.folio.service.auth;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dataimport.util.OkapiConnectionParams;
+import org.folio.service.auth.AuthClient.LoginCredentials;
+import org.folio.service.auth.PermissionsClient.PermissionUser;
+import org.folio.service.auth.UsersClient.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SystemUserAuthService {
+
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  public static final List<String> PERMISSIONS = Collections.unmodifiableList(
+    Arrays.asList(
+      "change-manager.jobexecutions.get",
```

Comment:
Thinking this list of permissions should be in a resource file

Reply:
Yeah, I'm not super happy with it there either.  I'll look at alternatives.

Reply:
Moved into an external file